src/library/Box/propTypes.js

```diff
@@ -8,7 +8,9 @@ export const spacingPropType = oneOfType([
 ]);
 
 export const boxPropTypes = {
+  borderRadius: oneOfType([string, arrayOf(string)]),
```

Comment:
I'm okay with breaking the alpha-convention here and simply listing all of our custom props at the top to more easily distinguish them.